test/project_types/extension/tasks/convert_server_config_test.rb

```diff
@@ -82,16 +82,18 @@ def test_empty_config_and_entry_fallback
         File.write(project_directory.join("src/index.js"), "")
 
         assert_nothing_raised do
-          Tasks::ConvertServerConfig.call(
+          result = Tasks::ConvertServerConfig.call(
             api_key: @api_key,
             context: @fake_context,
             hash: {},
             registration_uuid: @registration_uuid,
             store: @store,
             title: @title,
             tunnel_url: @tunnel_url,
-            type: @type
+            type: @type,
```

Comment:
As much as I like trailing commas, please revert this line, since we don't appear to have a linting rule in this project for trailing commas and this change is technically unrelated to the commit.